home-page.html

```diff
@@ -11,11 +11,63 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+    <script src="script.js" defer></script>
+    <script src="contributors.js" defer></script>
+    <link rel="stylesheet" href="pop-up-menu.css">
 </head>
-<body>
+<body id="home-page">
     <section class="background-image-wrapper">
         <div class="over-lay">
-            <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i></header>
+            <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i>
+                <div id="pop-up-menu-wrapper">
+                    <i class="fa-solid fa-xmark" id="cancel"></i>
+                <div class="pop-up-menu">
+                    <ul class=".pop-up-list">
+                        <li class="pop-up-li"><a class="page-link" href="home-page.html">Home</a></li>
+                        <li class="pop-up-li"><a class="page-link" href="index.html">About</a></li>
+                        <li class="pop-up-li">Program</li>
+                        <li class="pop-up-li">Join</li>
+                        <li class="pop-up-li">Sponsor</li>
+                    </ul>
+                </div>
+                </div></header>
+            <header class="about-desk-head">
+                <div class="sign-in-bar">
+                    <i class="fa-brands fa-facebook"></i>
+                    <i class="fa-brands fa-twitter"></i>
+                    <ul>
+                        <li><a href="">Logout</a></li>
+                        <li><a href="">Mypage</a></li>
+                        <li><a href="">Extras</a></li>
+                    </ul>
+                </div>
+                <div class="white-part">
+                    <div class="cc-image">
+                        <i class="fa-solid fa-closed-captioning"></i>
+                    </div>
+                    <div class="text">
+                        <p>
+                            CC GLOBAL
+                        </p>
+                        <p>
+                            Summit 2015
+                        </p>
+                        <p class="part-two dates">
+                            14th - 17th oct 2015 | Seoul, Korea
+                        </p>
+                    </div>
+                    <div class="menu-list">
+                        <ul class="menu-list-list">
+                            <li><a href="index.html">About</a></li>
+                            <li><a href="">Program</a></li>
+                            <li><a class="page-link" href="home-page.html">Home</a></li>
+                            <li><a href="">Sponsor</a></li>
```

Comment:
- [ ] In you home page in the desktop version I could see in your partner section the icons is too big I would higly recommend to decrease their height and width a little bit to match the given design as much as possible so please fix that 🙏🏻✨💯 and also the  height of the section here is too big please fix it 😊

- Yours
![image](https://user-images.githubusercontent.com/58553711/219921730-d22f09a4-f36c-4ae3-bb32-f1a350a27485.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/219921745-cab8eab5-2f9f-40a1-a95f-7447735ce49f.png)